### PR TITLE
Install tor from tor repository and add external tor as default

### PR DESCRIPTION
**Pull request by Emzy**
Fixes #replaceWithIssueNr, fixes #replaceWithIssueNr

Your PR description here.

### CODE REVIEW

**Comment by Emzy** [comment]
I tested it. Looks fine. Someone else need to test this.
You should see in the Bisq log: "[NetworkNode-8000] INFO  b.n.p.n.RunningTor: Connecting to running"
Instead of starting tor.

**Review by ripcurlx — DISMISSED** [review]
utACK

**Comment by ripcurlx** [comment]
@wiz @devinbileck Could you test this configuration? Thanks!

File: seednode/install_seednode_debian.sh [hunk 1]
```diff
@@ -69,6 +69,10 @@ sudo -H -i -u "${ROOT_USER}" git config --global advice.detachedHead false
 sudo -H -i -u "${ROOT_USER}" git clone --branch "${BISQ_REPO_TAG}" "${BISQ_REPO_URL}" "${ROOT_HOME}/${BISQ_REPO_NAME}"
 
 echo "[*] Installing Tor"
+sudo -H -i -u "${ROOT_USER}" wget -qO- https://deb.torproject.org/torproject.org/A3C4F0F979CAA22CDBA8F512EE8CBC9E886DDD89.asc | gp
+g --dearmor | tee /usr/share/keyrings/tor-archive-keyring.gpg >/dev/null
```
Comment (devinbileck) [inline seednode/install_seednode_debian.sh]: It seems this command is split over two lines.

**Review by devinbileck — DISMISSED** [review]
utACK
I have updated my seed nodes manually based on this configuration, and it is working.
I have not tested running the install from this shell script, though it looks fine to me aside from a minor issue with an unintentional line break.

File: seednode/install_seednode_debian.sh (hunk 1 above)
Comment (ripcurlx) [reply seednode/install_seednode_debian.sh]: ☝️ @Emzy Could you please comment on this? Thanks!

Comment (Emzy) [reply seednode/install_seednode_debian.sh]: Good catch. I fixed it.

**Review by devinbileck — DISMISSED** [review]

**Review by ripcurlx — DISMISSED** [review]
utACK - based on https://github.com/bisq-network/bisq/pull/6254#pullrequestreview-1014547569

**Comment by ripcurlx** [comment]
@Emzy Could you please sign your commits and force push the changes? Thanks!

**Comment by Emzy** [comment]
> @Emzy Could you please sign your commits and force push the changes? Thanks!

done

**Review by ripcurlx — APPROVED** [review]
utACK - based on https://github.com/bisq-network/bisq/pull/6254#pullrequestreview-1014547569

**Pull request merged**